src/_assets/css/_subscribe.scss

```diff
@@ -0,0 +1,163 @@
+.subscribe {
+  $required-input-color: #c53929;
+  $form-background-color: #e1eaf2;
+  $fields-border-color: #d8d8d8;
+  $submit-button-disabled-background: #c6c6c5;
```

Comment:
Would love if these were variables (colors)

Reply:
I decided to keep those as internal variables as they are not being used in other `.scss` documents.